### PR TITLE
PMM-12422-exporters-bind-push-mode

### PR DESCRIPTION
**Pull request by rasika-chivate**
For details see,

https://jira.percona.com/browse/PMM-12422

### CODE REVIEW

**Review by rnovikovP — CHANGES_REQUESTED** [review]
1) it's a little confusing to have different explanations for the same flag. is it really working differently based on agent or admin usage? 
2) "`0.0.0.0` address"  vs "127.0.0.1" explanation is very technical. Can we have a human-friendly one? To explain what this will cause for the user.

File: docs/details/commands/pmm-admin.md [hunk 1]
```diff
@@ -77,6 +77,9 @@ PMM communicates with the PMM Server via a PMM agent process.
 `--group=<group-name>`
 : Group name for external services. Default: `external`
 
+`--expose-exporter-address` (This flag is availble starting with PMM 2.41.0.)
+: When enabled, the exporter created for that service will use the `0.0.0.0` address.
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply docs/details/commands/pmm-admin.md:81]: there are good explanation in RN

**Review by BupycHuk — APPROVED** [review]

File: docs/details/commands/pmm-admin.md [hunk 2]
```diff
@@ -77,6 +77,9 @@ PMM communicates with the PMM Server via a PMM agent process.
 `--group=<group-name>`
 : Group name for external services. Default: `external`
 
+`--expose-exporter-address` (This flag is availble starting with PMM 2.41.0.)
```
Comment (idoqo) [inline docs/details/commands/pmm-admin.md:80]:
```suggestion
`--expose-exporter` (This flag is availble starting with PMM 2.41.0.)
```

We figured `--expose-exporter-address` felt too long, so we went with `--expose-exporter`

File: docs/details/commands/pmm-admin.md [hunk 3]
```diff
@@ -77,6 +77,9 @@ PMM communicates with the PMM Server via a PMM agent process.
...
+: If you enable this flag, any IP address on the local network and anywhere on the internet can access exporter endpoints. If the flag is disabled, exporter endpoints can be accessed only locally.
```
Comment (idoqo) [inline docs/details/commands/pmm-admin.md:81]:
```suggestion
: If you enable this flag, any IP address on the local network and anywhere on the internet can access exporter endpoints. If the flag is disabled/not present, exporter endpoints can be accessed only locally. The flag is disabled by default
```

File: docs/details/commands/pmm-agent.md [hunk 4]
```diff
@@ -64,6 +64,8 @@ Most options can be set via environment variables (shown in parentheses).
 | `--trace`                              | `PMM_AGENT_TRACE`                   | Enable trace output (implies `--debug`).
 | `-h`, `--help`                         |                                     | Show help (synonym for `pmm-agent help`).
 | `--version`                            |                                     | Show application version, PMM version, time-stamp, git commit hash and branch.
+| `--expose-exporter-address` (This flag is available starting with PMM 2.41.0.)| | If you enable this flag, any IP address on the local network and anywhere on the internet can access node exporter endpoints. If the flag is disabled, node exporter endpoints can be accessed only locally.
```
Comment (idoqo) [inline docs/details/commands/pmm-agent.md:67]:
```suggestion
| `--expose-exporter` (This flag is available starting with PMM 2.41.0.)| | If you enable this flag, any IP address on the local network and anywhere on the internet can access node exporter endpoints. If the flag is disabled, node exporter endpoints can be accessed only locally.
```

**Review by rnovikovP — APPROVED** [review]
LGTM, but with  @idoqo comment re default value

**Review by idoqo — APPROVED** [review]

**Pull request merged**